modules/conversion-center/classes/render/core-render.php

```diff
@@ -284,4 +284,41 @@ private function get_formatted_link_for_icon( array $icon ): string {
 
 		return $formatted_link;
 	}
+
+	private function get_layout_classnames(): void {
```

Comment:
This function should just `return $layout_classnames` (since it's a getter). Otherwise maybe call it `handle_class_names()` if you want all the logic to stay in here.

Reply:
@louiswol94 I settled on `build_layout_render_attribute` which is closer to what the function does, since it also handles ID